packages/cli/README.md

```diff
@@ -515,6 +516,25 @@ OPTIONS
 
 _See code: [src/commands/organizations/get.ts](https://github.com/Meeco/cli/blob/master/src/commands/organizations/get.ts)_
 
+## `meeco organizations:update`
+
+Modify a requested organization. The user who requested the organization can use this endpoint to modify the requested organization.
+
+```
+USAGE
+  $ meeco organizations:update
+
+OPTIONS
+  -a, --auth=auth                (required) [default: .user.yaml] Authorization config file yaml file (if not using the
+                                 default .user.yaml)
+
+  -e, --environment=environment  [default: .environment.yaml] environment config file
+
+  -o, --org=org                  (required) organization yaml file
```

Comment:
We may eventually need to do a consistency pass on these inputs - `organizations:create` uses `-c` for config but this uses `-o` for organization. Similar issues exist in other commands so might be best to just do a once-over at the end and improve it all at once.